kuttl-test.yaml

```diff
@@ -5,11 +5,16 @@ crdDir: ./tests/_build/crds/
 artifactsDir: ./tests/_build/artifacts/
 kindContainers:
   - local/jaeger-operator:e2e
+  - local/asserts:e2e
 commands:
   - command: kubectl create namespace jaeger-operator-system
   - command: kubectl apply -f ./tests/_build/manifests/01-jaeger-operator.yaml -n jaeger-operator-system
   - command: kubectl wait --timeout=5m --for=condition=available deployment jaeger-operator -n jaeger-operator-system
+  - command: kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/kind/deploy.yaml
+  - command: kubectl wait --namespace ingress-nginx --for=condition=ready pod --selector=app.kubernetes.io/component=controller --timeout=190s
```

Comment:
This was because my internet was failing and it takes sometime to pull the image hahaha. The instructions for deploy this ingress controller has 90s timeout. I think I can set it to that value.

Reply:
I suppose you could leave it or just change it to 180s.  Not a big deal.